docs/script.js

```diff
@@ -1,27 +1,250 @@
+    'featured image': 'sprites/projectPreview.svg',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    'link to live version': 'https://wandergithub.github.io/project-media/',
+    'link to source': 'https://github.com/wandergithub/project-media',
+  },
+  {
+    name: 'Tribute to Michael Jackson',
+    description: " Description text 2 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.",
+    'featured image': 'sprites/projectPreview.svg',
+    technologies: ['C++', 'C', 'c#'],
+    'link to live version': 'https://wandergithub.github.io/tribute-page/',
+    'link to source': 'https://github.com/wandergithub/tribute-page',
+  },
+  {
+    name: 'post 3',
+    description: "Description text 3 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.",
+    'featured image': 'sprites/projectPreview.svg',
+    technologies: ['python', 'Ruby on rails', 'Javascript'],
+    'link to live version': 'https://wandergithub.github.io/project-media/',
+    'link to source': 'https://github.com/wandergithub/project-media',
+  },
+  {
+    name: 'post 4',
+    description: "Description text 4 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.",
+    'featured image': 'sprites/projectPreview.svg',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    'link to live version': 'https://wandergithub.github.io/tribute-page/',
+    'link to source': 'https://github.com/wandergithub/tribute-page',
+  },
+  {
+    name: 'Multi-Post Storie',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    'featured image': 'sprites/img-placeholder.png',
+    technologies: ['css', 'html', 'Bootstrap', 'Ruby'],
+    'link to live version': 'https://wandergithub.github.io/project-media/',
+    'link to source': 'https://github.com/wandergithub/project-media',
+  },
+];
```

Comment:
Your implemented an array of objects to store JavaScript objects containing the information for each project 💪😎